listenbrainz/webserver/static/css/follow.less

```diff
@@ -0,0 +1,50 @@
+@purple: #353070;
+@white: #ffffff;
+
+#follow-button {
+  margin-left: 10px;
+  font-family: "Sintony";
+  color: @white;
+  background: @purple;
+  border: 2px solid @purple;
+  font-weight: bold;
+  padding: 5px 8px;
+  border-radius: 4px;
+  transition: color 0.42s, background 0.42s;
+  text-align: center;
+
+
+  &.icon-only {
+    > .text {
+      width: 0px;
+      margin-left: 0px;
+      display: inherit;
+      vertical-align: inherit;
+      transition: width 0.42s, margin-left 0.42s;
+      overflow-x: hidden;
+    }
+
+    &:hover, &:focus {
+      background: white;
+      color: #353070;
+
+      > .text {
+        min-width: 4.75em;
+        margin-left: 5px;
+      }
+    }
+  }
+
+  &.block {
+    width: 85px;
+
+    &.following {
```

Comment:
I think we should also add `&:hover` here, since otherwise the button does not show any indication of hovering (apart from the cursor change)

Reply:
It works to change following to unfollwo. Do we need to change styling for follow button too?

Reply:
I don't understand your comment. I don't see any `unfollow` class anywhere.

What I'm suggesting is a simple addition to match the `icon-only` style's
```less
&:hover, &:focus {
      background: white;
      color: #353070;
```
rule (line 27), that are missing on the `block` style.
I'm just suggesting to add `&:hover, &:focus` after `&.following` on this line (41) to have the same `background: white; color: #353070;` style for `block`.

Reply:
sorry for the confusion. done